web/templates/layout/app.html.eex

```diff
@@ -51,7 +51,7 @@
           <div class="dtc-ns w-50-l w-40-m v-mid">
             <%= transform_html @banner, @conn, class: "underline lm-white dib", id: "feedback-link" %>
           </div>
-          <div class="dtc-ns dn tr v-mid">
+          <div class="dtc-ns dn tr v-mid banner_buttons">
```

Comment:
If we need to add another class to make them 'disappear' at a smaller larger breakpoint, how are we making them disappear right now in staging? I couldn't see it by inspecting the button classes, just curious 👍